pkg/receive/handler.go

```diff
@@ -196,10 +196,20 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 
 	ins := extpromhttp.NewNopInstrumentationMiddleware()
 	if o.Registry != nil {
+		var buckets = []float64{0.001, 0.005, 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1, 0.25, 0.5, 0.75, 1, 2, 3, 4, 5}
+
+		const bucketIncrement = 2.0
```

Comment:
just curious if linear incremental works well if, let say, forwardTimeout is something in the area of 1 ... 10 minutes :thinking:

Reply:
It will never be higher than 20 - 30 seconds so I think it's completely fine to have more buckets here.